appcheck/firebase-appcheck/src/main/java/com/google/firebase/appcheck/internal/DefaultFirebaseAppCheck.java

```diff
@@ -110,7 +112,8 @@ public void resetAppCheckState() {
   public void addAppCheckTokenListener(@NonNull AppCheckTokenListener listener) {
     checkNotNull(listener);
     appCheckTokenListenerList.add(listener);
-    tokenRefreshManager.onListenerCountChanged(appCheckTokenListenerList.size());
+    tokenRefreshManager.onListenerCountChanged(
```

Comment:
I'm assuming that we're intentional about the fact that external listeners cause auto-refreshing, right?

Reply:
Yeah, I think that's what makes sense, but happy to discuss if you disagree!